alma/views/templates/hook/displayProductActions.tpl

```diff
@@ -20,8 +20,7 @@
  * @copyright 2018-2024 Alma SAS
  * @license   https://opensource.org/licenses/MIT The MIT License
  *}
-{if isset($layout)}
-    <div class="alma-widget-insurance" id="alma-widget-insurance-product-page" data-alma-insurance-settings="{$settingsInsurance}" style="height: 0px" >
-        <iframe id="product-alma-iframe" src="{$iframeUrl}"></iframe>
-    </div>
-{/if}
+<div id="alma-product-details" data-product-details="{$productDetails}" style="display:none;"></div>
```

Comment:
{if isset($layout)} not necessary ?

Reply:
@Francois-Gomis No because, Ravate use Creative Element and Creative Element doesn't set layout.
And this fix has been done for PS 1.6 but we don't have insurance in PS 1.6

Reply:
And for others 1.7 without Creative elements ?

Reply:
@Francois-Gomis The template is in a hook, if the hook isn't load we don't display the template. We don't need to add a condition about layout. Like the BNPL Widget